src/pages/workspace/WorkspaceMembersPage.tsx

```diff
@@ -146,54 +138,54 @@ function WorkspaceMembersPage(props) {
 
     useEffect(() => {
         validateSelection();
-    }, [props.preferredLocale, validateSelection]);
+    }, [preferredLocale, validateSelection]);
 
     useEffect(() => {
-        if (removeMembersConfirmModalVisible && !_.isEqual(accountIDs, prevAccountIDs)) {
+        if (removeMembersConfirmModalVisible && JSON.stringify(accountIDs.sort()) !== JSON.stringify(prevAccountIDs.sort())) {
             setRemoveMembersConfirmModalVisible(false);
         }
         setSelectedEmployees((prevSelected) => {
             // Filter all personal details in order to use the elements needed for the current workspace
-            const currentPersonalDetails = filterPersonalDetails(props.policyMembers, props.personalDetails);
+            const currentPersonalDetails = filterPersonalDetails(policyMembers, personalDetails);
             // We need to filter the previous selected employees by the new personal details, since unknown/new user id's change when transitioning from offline to online
-            const prevSelectedElements = _.map(prevSelected, (id) => {
-                const prevItem = lodashGet(prevPersonalDetails, id);
-                const res = _.find(_.values(currentPersonalDetails), (item) => lodashGet(prevItem, 'login') === lodashGet(item, 'login'));
-                return lodashGet(res, 'accountID', id);
+            const prevSelectedElements = prevSelected.map((id) => {
+                const prevItem = prevPersonalDetails?.id;
+                const res = Object.values(currentPersonalDetails).find((item) => prevItem?.login === item?.login);
+                return res?.accountID ?? id;
             });
-            return _.intersection(prevSelectedElements, _.values(PolicyUtils.getMemberAccountIDsForWorkspace(props.policyMembers, props.personalDetails)));
+            return [prevSelectedElements, Object.values(PolicyUtils.getMemberAccountIDsForWorkspace(policyMembers, personalDetails))].reduce((a, b) => a.filter((c) => b.includes(c)));
```

Comment:
Maybe let's try to replace `a`, `b`, `c` with more specific naming

Reply:
what's a default value for reduce's accumulator?

Reply:
In the [docs](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_intersection) that I used to migrate this function, there is no default value, so I guess it's not necessary.